colossus/src/main/scala/colossus/core/Server.scala

```diff
@@ -299,7 +299,7 @@ private[colossus] class Server(io: IOSystem, serverConfig: ServerConfig,
 
 
   def binding(router: ActorRef): Receive = alwaysHandle orElse {
-    case DelegatorBroadcast(message) => router ! akka.routing.Broadcast(Worker.DelegatorMessage(me, message))
+    case DelegatorBroadcast(message) => router ! akka.routing.Broadcast(Worker.InitializerMessage(me, message))
```

Comment:
This case class needs to be changed.  There's also a method in `ServerRef` called `delegatorBroadcast`, that method should _not_ be removed, but marked as deprecated, and then also create it's replacement `initializerBroadcast`.